lib/connection.js

```diff
@@ -20,6 +21,7 @@ class Connection extends EventEmitter {
     this.buffer = [];
     this.requestBuffer = [];
     this.requested = 0;
+    this.requestCallbacks = {};
```

Comment:
If we specified that the minimum `nodejs` version for this module is `v14`, then we can use es6 `Map` as a dictionary instead of an `object`.

The `Map` will be more effective when items are constantly added and removed.